src/main/java/com/zendesk/maxwell/schema/ddl/MysqlParserListener.java

```diff
@@ -16,7 +16,7 @@
 import org.slf4j.LoggerFactory;
 
 public class MysqlParserListener extends mysqlBaseListener {
-    final Logger LOGGER = LoggerFactory.getLogger(MysqlParserListener.class);
+		final Logger LOGGER = LoggerFactory.getLogger(MysqlParserListener.class);
```

Comment:
Yeah, a find/replace all was not the best idea. I switch to Intellij, was using Atom.